component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/service/ProducerFinderImpl.java

```diff
@@ -76,7 +91,7 @@ static class InputIterator implements Iterator<Object> {
 
         private Object nextObject;
 
-        private boolean init = false;
+        private boolean init;
```

Comment:
Why remove explicit initialize .

Reply:
matter of taste... default value for `boolean` primitive is `false`.